core/utils/hashmap.c

```diff
@@ -1,135 +0,0 @@
-/**
- * @author Peter Donovan (peterdonovan@berkeley.edu)
```

Comment:
Why are we getting rid of this?

Reply:
Because it has not been used, and because we can always bring it back if we want. I added it because I thought it might be used imminently, but it turned out not to be necessary.

Reply:
Why don't we keep it? It's a data structure, so it's of general utility.

Reply:
Okay, sure. I'll bring it back.